GoogleSignIn.podspec

```diff
@@ -29,6 +29,7 @@ The Google Sign-In SDK allows users to sign in with their Google account from th
     'Security'
   ]
   s.ios.framework = 'UIKit'
+  s.osx.framework = 'AppKit'
```

Comment:
Let's update the test_spec below to get unit tests running on macOS via pod lib lint.

Reply:
I believe you've resolved this?  pod lib lint seems to be running tests on both platforms now.

Reply:
Yes, this is done.